cob_people_detection/ros/src/face_recognizer_node.cpp

```diff
@@ -518,7 +518,7 @@ void FaceRecognizerNode::facePositionsCallback(const cob_perception_msgs::ColorD
 				// set label
 				det.label = identification_labels[head][face];
 				// set origin of detection
-				det.detector = "face";
+				det.detector = ros::this_node::getName();
```

Comment:
mhh, ok.

there are three fields in the message that are unclear to me:
- detector
- label
- id
Can you explain for what they are used?

I'm searching for a field that allows me to distinguish where a messages comes from in a multi detector setup.